docs/topics/installing-mta-disconnected-environment.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="installing-mta-disconnected-environment_{context}"]
+= Installing {ProductShortName} on a disconnected environment
+
+On a connected device, first download and save the {ProductShortName} binary. Then download and save the Podman images, the {ProductShortName} Podman image and the provider image that you need.
+
+* Download the required {ProductShortName} CLI binary from the link:https://developers.redhat.com/products/mta/download[{ProductName} Red Hat Developer page]:
```

Comment:
missing a space after the colon

Reply:
It looks like it renders ok

![image](https://github.com/user-attachments/assets/fa65554e-1ef4-4094-afbc-1df99ba65284)